src/ui/public/styles/pagination.less

```diff
@@ -29,6 +29,9 @@ paginate {
 
           a {
             text-decoration: none;
+            background-color: @white;
```

Comment:
Yes, according to the spec, the pagination style is consistent through the different views.